README.md

```diff
@@ -1,3 +1,4 @@
+![Nuget](https://img.shields.io/nuget/v/Microsoft.CST.OSSGadget.Shared)

```

Comment:
```suggestion
[![Nuget](https://img.shields.io/nuget/v/Microsoft.CST.OSSGadget.Shared)](https://www.nuget.org/packages/Microsoft.CST.OSSGadget.Shared)
```
I think you want to do something like this. If you don't have an extra link markdown around the badge it is clickable but it will just open the badge icon in the browser.